### ISSUE TITLE
Twig type hints don't seem to work with completion for me

### ISSUE THREAD

**Issue by bravoman** [klesun/deep-assoc-completion#165]
I might be doing something wrong but if I use the following type hint in twig:

```twig
{# @var client array = Magic::dbRow('clients') #} 
{{ client. }} <!-- press CTRL+SPACE for typehints -->
```

This doesn't seem to give me completion. Am I doing something wrong or is this not implemented (yet)?

Source for the twig variable type hint support in PHPStorm: https://stackoverflow.com/a/30997797/839332

I found #119 but this seems to deliver me the option to define a template function that is gives the source data for the full  twig template (all variables) as opposed to just the one variable I'm trying to describe.

**Comment by klesun** [comment]
Hi. Thanks for the reference, did not know such format already existed in laravel/symfony plugin.

There is no `=` support for particular vars in the deep-assoc plugin, only the #119 you mentioned which is only used to describe whole structure.

I'll take a look if I can add the `=` support for vars without much pain...

**Comment by bravoman** [comment]
Cool, that would be very nice. Otherwise I guess I still have options 👍🏻

**Comment by klesun** [comment]
Teaser:
![image](https://user-images.githubusercontent.com/5202330/92533860-0ab7d200-f23c-11ea-8aac-a77ae398c585.png)

**Comment by klesun** [comment]
Released in [2020.09.09.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/versions/stable/96492)

Please, let me know if you encounter any issues

**Comment by bravoman** [comment]
🤤 ...
